rest/src/main/java/discord4j/rest/route/Routes.java

```diff
@@ -256,6 +256,19 @@
     public static final Route<Void> MESSAGE_SUPPRESS_EMBEDS = Route.post(
             "/channels/{channel.id}/messages/{message.id}/suppress-embeds", Void.class);
 
+    /**
+     * Enable/disable suppression of embeds on a Message. This endpoint requires the 'MANAGE_MESSAGES' permission to
+     * be present for the current user.
+     * <p>
+     * Returns a 204 empty response on success. Fires a Message Update Gateway event.
+     *
+     * @see <a href="https://discordapp.com/developers/docs/resources/channel#suppress-message-embeds">
+     *         https://discordapp.com/developers/docs/resources/channel#suppress-message-embeds</a>
+     */
+    @Experimental
+    public static final Route<Void> CROSSPOST_MESSAGE = Route.post(
```

Comment:
The documentation does not correspond to this Route

Reply:
There is no official documentation, though from my tests its an empty POST request and it returns a Message object

Reply:
Ups a mistake when copy this code...
About the docs: https://github.com/discord/discord-api-docs/issues/1510#issuecomment-613719406